hack/ocm-minikube-ramen.sh

```diff
@@ -0,0 +1,360 @@
+#!/bin/sh
+# shellcheck disable=1090,2046,2086
+set -x
+set -e
+trap 'set -- ${?}; trap - EXIT; eval ${exit_stack}; echo exit status: ${1}' EXIT
+trap 'trap - ABRT' ABRT
+trap 'trap - QUIT' QUIT
+trap 'trap - TERM' TERM
+trap 'trap - INT' INT
+trap 'trap - HUP' HUP
+exit_stack_push()
+{
+	{ set +x; } 2>/dev/null
+	exit_stack=${*}\;${exit_stack}
+	set -x
+}
+exit_stack_push unset -v exit_stack
+exit_stack_push unset -f exit_stack_push
+exit_stack_pop()
+{
+	IFS=\; read -r x exit_stack <<-a
+	${exit_stack}
+	a
+	eval ${x}
+	unset -v x
+}
+exit_stack_push unset -f exit_stack_pop
+rook_ceph_branch_checkout()
```

Comment:
This function would be removed, right? As the rook scripts are checked in...

Reply:
Yes!  It has been removed.